.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dockerFile": "Dockerfile",
+  "extensions": [
+    "ionide.ionide-fsharp"
+  ],
+  "settings": {
+    "[csharp]": {
+      "editor.useTabStops": false,
+      "editor.tabSize": 4
+    },
+
+    "[fsharp]": {
+      "editor.useTabStops": false,
+      "editor.tabSize": 4
+    },
+
+    "FSharp.useSdkScripts": true
```

Comment:
What does it means? `"FSharp.useSdkScripts": true`

Reply:
It's a configuration setting from `ionide`, it causes `ionide` to use `dotnet fsi` instead of regular `fsi` for executing F# scripts.